packages/streamr-logger/test/unit/StreamrLogFormatterTest.cpp

```diff
@@ -0,0 +1,103 @@
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <streamr-logger/detail/StreamrLogFormatter.hpp>
+using namespace streamr::logger::detail; // NOLINT
```

Comment:
can linter be set to not need `// NOLINT` here? Or else add explanation why this one is exceptional.